txpipe/twopoint.py

```diff
@@ -317,50 +292,140 @@ def write_output(self, source_list, lens_list, meta, results):
 
         # Add the covariance.  There are several different jackknife approaches
         # available - see the treecorr docs
-        cov = treecorr.estimate_multi_cov(comb, self.config["var_method"])
+        print(comb[0])
```

Comment:
and remove these extra print statements.

Reply:
Done.